apps/src/code-studio/pd/application/teacher1920/Teacher1920Application.jsx

```diff
@@ -22,6 +22,31 @@ export default class Teacher1920Application extends FormController {
 
   static sessionStorageKey = 'Teacher1920Application';
 
+  /**
+   * @override
+   */
+  componentWillMount() {
```

Comment:
Nonblocking right now, but `componentWillMount` [is deprecated in React 16.3](https://medium.com/@baphemot/whats-new-in-react-16-3-d2c9b7b6193b) and we should be moving away from using it.  It should be okay to do this in `componentDidMount` eventually.

Reply:
Discussed offline. We keep it in `componentWillMount` for now because `SchoolAutocompleteDropdown` needs `state` info before being rendered.